packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -121,7 +121,11 @@ export class DialogBase extends React.Component<IDialogProps, {}> {
     const dialogContentProps: IDialogContentProps = {
       ...DefaultDialogContentProps,
       ...this.props.dialogContentProps,
-      draggableHeaderClassName: dialogDraggableClassName
+      draggableHeaderClassName: dialogDraggableClassName,
+      titleProps: {
+        id: this.props.dialogContentProps?.titleId || this._defaultTitleTextId,
+        ...this.props.dialogContentProps?.titleProps
+      }
```

Comment:
Some snapshot tests may be good here with varying values of `titleId` and `titleProps`, including `undefined` values. That should help prevent regressions in the future.

Reply:
(Also be good to test the tests by temporarily commenting out the change, verifying they fail, and then are fixed with your change. 😄 )

Reply:
sure, let me add more coverage